lib/doorkeeper/rails/helpers.rb

```diff
@@ -50,11 +50,11 @@ def doorkeeper_error
         end
       end
 
-      def doorkeeper_render_options
+      def doorkeeper_render_options(error)
         if doorkeeper_invalid_token_response?
-          doorkeeper_unauthorized_render_options
+          doorkeeper_unauthorized_render_options(error)
         else
-          doorkeeper_forbidden_render_options
+          doorkeeper_forbidden_render_options(error)
```

Comment:
This is the method call line, not the declaration line. Should we still use ```error: error``` as well?